php/blocks/controls/class-user.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * User control.
+ *
+ * @package   Block_Lab
+ * @copyright Copyright(c) 2018, Block Lab
+ * @license http://opensource.org/licenses/GPL-2.0 GNU General Public License, version 2 (GPL-2.0)
+ */
+
+namespace Block_Lab\Blocks\Controls;
+
+/**
+ * Class User
+ */
+class User extends Control_Abstract {
+
+	/**
+	 * Control name.
+	 *
+	 * @var string
+	 */
+	public $name = 'user';
+
+	/**
+	 * User constructor.
+	 */
+	public function __construct() {
+		parent::__construct();
+		$this->label = __( 'User', 'block-lab' );
+	}
+
+	/**
+	 * Register settings.
+	 *
+	 * @return void
+	 */
+	public function register_settings() {
+		$this->settings[] = new Control_Setting(
+			array(
+				'name'     => 'placeholder',
+				'label'    => __( 'Placeholder Text', 'block-lab' ),
+				'type'     => 'text',
+				'default'  => __( 'Enter a username', 'block-lab' ),
```

Comment:
This is the first time we are suggesting a default Placeholder. Just wanted to make sure we want to do that. @RobStino thoughts?

![screen shot 2019-03-03 at 7 26 51 pm](https://user-images.githubusercontent.com/1097667/53693304-694ff480-3dea-11e9-8490-e7a5687ec2cd.png)

Reply:
Now that there's help text in https://github.com/getblocklab/block-lab/pull/233/commits/d80465db25a7c78261801ab85f92e9675cf06b13, maybe removing the default placeholder would be good.

Reply:
Great. Let's keep it consistent then. I don't mind adding default placeholder text (or even help text), but if we do we should do it for all controls for consistency.

Reply:
Sounds good, 7bbb0ed97d923dc921218821df0a50eb1cbf0e53 removes the placeholder's default value.